swagger.go

```diff
@@ -62,6 +70,7 @@ func EchoWrapHandler(configFns ...func(c *Config)) echo.HandlerFunc {
 		DeepLinking:  true,
 		DocExpansion: "list",
 		DomID:        "#swagger-ui",
+		RedirectUrl:  nil,
```

Comment:
This is really not necesary. Why do you need to initialize a property that's default nill with a nil value?